main.go

```diff
@@ -108,9 +154,8 @@ func main() {
 		Development: true,
 	}
 	opts.BindFlags(flag.CommandLine)
```

Comment:
then here we would bind flags with the args from the config map.

Reply:
Don't need touch the BindFlags which is just add those logger options to command line.